pengwin-setup.d/uninstall/nim.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+source $(dirname "$0")/uninstall-common.sh
+
+function main()
+{
+
+local nim_conf="/etc/profile.d/nim.sh"
+
+echo "Uninstalling nim"
+
+echo "Removing ~/.choosenim/"
```

Comment:
don't need echo statements here in the uninstall script, the common functions i've provided in uninstall-common already provide output :)